src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -3983,5 +3984,62 @@ public void AssertNotDisposed()
             if (_disposeOne.Disposed)
                 ThrowObjectDisposed();
         }
+
+        public int Dump(string path, Action<IOperationProgress> onProgress)
+        {
+            if (Directory.Exists(path) == false)
+                Directory.CreateDirectory(path);
+
+            using (var tx = _environment.ReadTransaction())
```

Comment:
I'm wondering if we should not put `CurrentlyInUse` here?

Reply:
Yes, I think that would be good

Reply:
done